utils/build-script-impl

```diff
@@ -2516,6 +2520,40 @@ for host in "${ALL_HOSTS[@]}"; do
                     LIBICU_BUILD_ARGS=()
                 fi
 
+                if [[ ! "${SKIP_BUILD_LIBXML2}" ]]; then
+                    BASE_INSTALL_DIR="$(get_host_install_destdir ${host})"
+                    LIBXML2_HEADERS="${BASE_INSTALL_DIR}/usr/include/libxml2"
+                    LIBXML2_BUILD_ARGS=(-DLIBXML2_INCLUDE_DIR:PATH="$LIBXML2_HEADERS")
+                    # Find static library
+                    LIBXML2_LIB=$(find "${BASE_INSTALL_DIR}"  -name 'libxml2.a')
```

Comment:
Please don't use `find` for this.  You know it should be at `${BASE_INSTALL_DIR}/usr/lib/libxml2.a`, just write it out.

Reply:
Done